src/Components/Modal/style.module.scss

```diff
@@ -0,0 +1,17 @@
+@import "../../colors.module.css";
+@import "../../fonts.module.css";
+
+.container {
+  border: 2px solid var(--LIGHT_GRAY);
+  border-radius: 8px;
+}
+.icon {
+  padding: .5rem !important;
```

Comment:
@timtran007 lets see if we can get this working without `!important`, instead of this there is a prop you can pass to your component called `cssModule` feel free to do a find-in-file to see examples of how it's been implemented in our code

docs: https://reactstrap.github.io/?path=/docs/components-layout--props